.gitignore

```diff
@@ -55,3 +55,31 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Jupyter and iPython notebook checkpoints
+*.ipynb_checkpoints
+
+# Spyder app workspace config file
+.spyderworkspace
+
+# PyCharm environment files
+.idea/
+
+# VSCode environment files
+.vscode/
+*.code-workspace
+
+# Apple OSX
+*.DS_Store
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# pythonenv
+pythonenv*/
+
+# JS dependencies
```

Comment:
These are specific to the Mesa repo only.

Reply:
Right, will delete